runtime/expr/ztests/unary-minus.yaml

```diff
@@ -0,0 +1,47 @@
+zed: yield -this
+
+input: |
```

Comment:
Add cases for `null`, `null(uint8)`, etc.?

Reply:
oopsies